cnxdb/archive-sql/get-featured-links.sql

```diff
@@ -17,11 +17,11 @@ FROM (SELECT
     a.html AS abstract,
     '/resources/' || f.sha1 AS "resourcePath",
```

Comment:
This is going to fail hard - `f` is now `featured_books` when before it was `files`.

Reply:
Suggested resolving this by actually storing the cover fileid here, and joining to files to get the sha1